src/lib/utils.ts

```diff
@@ -290,10 +297,11 @@ export function isFulfilled<T>(
 
 // Get the explorer link of an address, eg. Etherscan.
 export function explorerLink(addr: string, wallet: Wallet): string {
```

Comment:
We should rename the `addr` attribute to `addrOrTx`.

Reply:
1e4ebf0